app/include/dt-bindings/zmk/mouse.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+#pragma once
+
+/* Mouse press behavior */
+/* Left click */
+#define MB1 (0x01)
+#define LCLK (MB1)
+
+/* Right click */
+#define MB2 (0x02)
+#define RCLK (MB2)
+
+/* Middle click */
+#define MB3 (0x04)
+#define MCLK (MB3)
+
+#define MB4 (0x08)
+
+#define MB5 (0x10)
+
+#define MB6 (0x20)
+
+#define MB7 (0x40)
+
+#define MB8 (0x80)
+
+/* Mouse move behavior */
+
+#define MOVE_UP (0x0000FFFF)
```

Comment:
I think we should make the MOVE_UP take an argument to control the speed (e.g. MOVE_UP(100), MOVE_LEFT(50), etc)

Later we can make something that interprets/translates this speed based on an acceleration curve, but I think that doesn't have to be part of this PR.

Reply:
Maybe also a `MOUSE_MOVE(HORIZONTAL, VERTICAL)` macro which can combine a horizontal and vertical move in one go.

Reply:
I was thinking about these, I initially thought we'd have to decide between one-way (horizontal-only or vertical-only) or two-way macros, but there's no reason not to implement both.

Reply:
Is there an upper bound on mouse buttons or is MB8 arbitrarily chosen?

Reply:
16 buttons, each bit in a `uint_16` is basically a flag. MB8 was chosen basically arbitrarily, I suppose we should extend it to MB16 (although we haven't tested buttons 9-16).

Reply:
Added buttons up to MB16, as well as configurable movement and wheel macros. Please pay attention to the comments: we need some testing to determine usable value ranges.

Reply:
Tested the added mouse buttons, haven't been able to get 9-16 to work (tried adjusting the Usage Maximum in `hid.h`, didn't help). Reverted the new macros. Buttons 1-8 work.